src/demo/components/CodeSnippetPreview.css

```diff
@@ -73,4 +73,9 @@
     justify-content: center; 
     gap: 56px; 
   }
-  
\ No newline at end of file
+  
+  @media only screen and (min-width: 375px) and (max-width: 767px) {
+    .display-none{
```

Comment:
maybe better to call this `mobile-display-none` to make it more clear

Reply:
we can use this for mobile plus web like when we add logout icon on top view of mobile may be we can hide it on web view.